commcare_connect/opportunity/visit_import.py

```diff
@@ -287,7 +287,7 @@ def get_status_by_completed_work_id(dataset):
     invalid_rows = []
     for row in dataset:
         row = list(row)
-        work_id = row[work_id_col_index]
+        work_id = str(row[work_id_col_index])
```

Comment:
What is this change for?

Reply:
This is the fix for the broken payment verification import due to the work_id being a string when the import file was CSV and an integer when the import file was XLSX. 😢